src/navigations/HomeBottomTabs.js

```diff
@@ -113,18 +113,18 @@ function HomeBottomTabs() {
     if (route.name === 'Profile') label = 'Profile';
 
     const bottomBarExtendedStyle = {
-      color: nav?.isFocused() ? activeColor : COLORS.blackgrey
+      color: nav?.isFocused() ? activeColor : COLORS.gray400
     };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color change from `COLORS.blackgrey` to `COLORS.gray400` for the inactive state of the navigation tab seems fine. However, it's important to ensure that this color change is consistent across the application and doesn't affect the readability or user experience.



<!-- This is an auto-generated comment by OSS CodeRabbit -->